src/rules/cyclomaticComplexityRule.ts

```diff
@@ -105,6 +105,7 @@ function walk(ctx: Lint.WalkContext<{ threshold: number }>): void {
 function increasesComplexity(node: ts.Node): boolean {
     switch (node.kind) {
         case ts.SyntaxKind.CaseClause:
+            return (node as ts.CaseClause).statements.length > 0;
```

Comment:
There's only one possible path through that code, so I don't think it should increase complexity.